crud.test.js

```diff
@@ -52,3 +63,136 @@ describe('deleteAndRemoveTask', () => {
     expect(document.querySelector('.todo-item')).toBeFalsy();
   });
 });
+
```

Comment:
- [ ] 8 of your tests failed. It'd be great if all of them can be pass the test to continue. 😉

Reply:
@eri8-9 

Thanks for the review! Can you provide us a screenshot of the failed tests? We're green across the board here so we're not sure what you might be experiencing. Can you provide some more context so we can troubleshoot? This is how ours looks:

![image](https://user-images.githubusercontent.com/92178070/161153087-38881bc6-cf04-4995-8ac1-596c2c2e9f48.png)

Reply:
@eri8-9 

Thanks this is helpful and I see the issue you're experiencing. It looks like the version of node that you're running is not recognizing the latest syntax which is why it is failing on the tests using .at(). Our version  of Node (after calling "node --version" in the terminal) is 16.14.2. Can you update your node and try again? I believe that is the issue. Some documentation on .at() can be found here if you need to validate that it is valid syntax: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at